app/pandas.ipynb

```diff
@@ -234,10 +300,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "outputs": [],
+   "execution_count": 19,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "0    6.0\n1   -6.0\n2    0.0\ndtype: float64"
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] - df[\"Age\"]"
```

Comment:
You need to calculate the absolute difference between Age and Average age.
https://en.wikipedia.org/wiki/Absolute_difference